### DISCUSSION TITLE
Aircraft trails

### DISCUSSION THREAD

**Discussion by Sky173** [albar965/littlenavmap#1172] [Q&A]
I'm curious if there is a way to add AI/multiplayer aircraft trails. I don't see a setting in the menus unless I missed it. If not, that would be a nice idea to have, esp if people like plane-spotting from the ground while watching LittleNavMap at the same time. Thanks.

**Comment by albar965** [comment]
> I'm curious if there is a way to add AI/multiplayer aircraft trails. I don't see a setting in the menus unless I missed it. If not, that would be a nice idea to have, esp if people like plane-spotting from the ground while watching LittleNavMap at the same time. Thanks.

There is no option for AI trails, only for the user aircraft. I may add this in a future version, but not anytime soon.

Alex
